general/hash.hpp

```diff
@@ -136,6 +136,9 @@ class HashTable : public BlockArray<T>
    /// Write details of the memory usage to the mfem output stream.
    void PrintMemoryDetail() const;
 
+   /// Print hash table statistic for debugging purposes.
```

Comment:
This documentation is not clear, what are these statistics?

Reply:
Improved.